numpy/random/mtrand.pyx

```diff
@@ -3628,7 +3628,8 @@ cdef class RandomState:
             Mean of the N-dimensional distribution.
         cov : 2-D array_like, of shape (N, N)
             Covariance matrix of the distribution. It must be symmetric and
-            positive-semidefinite for proper sampling.
+            positive-semidefinite for proper sampling. It must be a factor
+            of the covariance matrix if the `mode` argument is set to 'factor'.
```

Comment:
Scrolling all the way down to the example I see this is expected to be a cholesky factor which is a bit specific. What if someone wanted to pass in factors for a QR or SVD (because perhaps cholesky blows up for their matrix).

Maybe the api should be:
```
mode : { 'eigh', 'svd', 'cholesky'}
precomputed : bool
    If `True` the `cov` argument is interpreted as the pre-computed factor for the
    method given by the `mode` argument
```
```

Reply:
Of course this may require passing in a dictionary of factors:
```python
multivariate_normal(mean, cov={'Q': Q, 'R': R}, mode='eigh', precomputed=True)
multivariate_normal(mean, cov={'U': u, 'S': s, 'V': v}, mode='svd', precomputed=True)
```
...which maybe isn't a *great* api?

Perhaps @eric-wieser's suggestion of a separate method is cleaner?
```python
multivariate_normal.from_factor(mean, **factors, method='svd')
```

Reply:
it is meant to work with any other factor. The check for upper triangular structure is so that the right multiplication can be done if the upper tringular factor of cholesky is used. Wouldn't it be better to require the user to supply the full factor matrix instead of its components? Im not sure how i'd elegantly  implement `multivariate_normal.from_factor` given that python classes do not allow creation of method attributes.